FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureBlobJobStore.cs

```diff
@@ -132,28 +138,28 @@ public async Task<bool> UpdateJobAsync(Job job, CancellationToken cancellationTo
                     job.ToStream(),
                     cancellationToken: cancellationToken);
 
+                _logger.LogInformation("Update job '{job.Id}' successfully.", job.Id);
                 return true;
             }
             catch (Exception ex)
             {
-                _logger?.LogWarning(ex, "Failed to update job '{job.Id}'.", job.Id);
-                return false;
+                _logger.LogError(ex, "Failed to update job '{job.Id}'", job.Id);
+                throw;
```

Comment:
If we are throwing exceptions here, it looks we don't need to return a bool as return value?

Reply:
Good idea. Fixed